README.md

```diff
@@ -73,6 +73,13 @@ Requires build-essential tools and SDL2 development libraries to be already inst
 ~/hdzero-gogglei/build$ ./HDZGOGGLE
 ```
 
+## Emulator Keys
```

Comment:
Not sure if this deserves it's own bullet... perhaps just make it part of the Build Emulator section as a note.